### PR TITLE
🤖 Consolidate install steps for all github actions

### PR DESCRIPTION
**Pull request by fwkoch**
This is trying out a "composite action" so install steps are not duplicated across ci / release actions.

### CODE REVIEW

File: .github/actions/install/action.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    default: '18'
+runs:
+  using: 'composite'
+  steps:
+    - name: Install libxml
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libxml2-utils
+      shell: bash
+    - name: Setup Node.js 18.x
```
Comment (rowanc1) [inline .github/actions/install/action.yml:16]:
```suggestion
    - name: Setup Node.js
```

**Review by rowanc1 — COMMENTED** [review]
Looks great!

**Comment by rowanc1** [comment]
I am consistently seeing timeout failures on the execute tests. Can we add a larger timeout flag, not sure if that is what is happening:
https://github.com/executablebooks/mystjs/blob/main/packages/myst-execute/tests/run.spec.ts#L101

**Comment by fwkoch** [comment]
Hmm, yeah, successful tests take right about the same amount of time as the timeout:
image

I'll try bumping the timeout, but I wonder if there's something we can do to speed up the jupyter loading (launch the server outside of running the tests or something...?).

**Pull request merged**